test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/Program.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.TestPlatform.CommunicationUtilities.PlatformTests
+{    
+    public static class Program
```

Comment:
Yes, for every test project that targets netcoreapp. We don't include NET.Test.Sdk directly.